app/src/main/java/com/ably/tracking/demo/publisher/api/DeliveryServiceApi.kt

```diff
@@ -0,0 +1,17 @@
+package com.ably.tracking.demo.publisher.api
+
+import retrofit2.http.GET
+import retrofit2.http.Header
+
+interface DeliveryServiceApi {
```

Comment:
I would personally remove 'Service' from this name, but please keep it if you think it's better this way

Reply:
I wanted to reflect the path chosen by Quintin on the backend side:
(...).net/**deliveryService**/orders
But if you think this is unnecessary I can remove that part

Reply:
I don't mind at all. It's just 'service' and 'api' are very close concepts, I thought the class name can be shortened. But I see that you wanted to be consistent with backend side which makes sense.